### PR TITLE
Error in your code.

### PR DESCRIPTION
**Pull request by birju8kumar**
I have resolved the error for you.

### CODE REVIEW

**Comment by kumarbirju8** [comment]
I did not understand.

**Comment by birju8kumar** [comment]
Just verify and pull the changes.

File: birju [hunk 1]
```diff
@@ -0,0 +1,2 @@
+I have added few lines in birju file.
+And the problem is resolved.
```
Comment (kumarbirju8) [inline birju]: I dont want this line.
Please remove it.

Comment (birju8kumar) [reply birju]: ok.

Comment (birju8kumar) [reply birju]: check now.

Comment (kumarbirju8) [reply birju]: All right, thanks.

**Pull request merged**